resources/case_studies/Insurance/Bid.obs

```diff
@@ -1,10 +1,10 @@
-main contract Bid {
+main resource contract Bid {
     int cost;
 
-    Bid(int c) {
+    owned Bid(int c, int time) {
```

Comment:
Change the name of "time" to specify what time it is. Expiration date?

Reply:
Also, do you have a plan for how to really express this on the blockchain? It appears there may be no equivalent of oraclize on Fabric…?